### PR TITLE
Jammy debian/cloud-init.templates: enable Akamai by default

### PR DESCRIPTION
**Pull request by aciba90**
## Additional context

https://github.com/canonical/cloud-init/issues/4362
Continuation of https://github.com/canonical/cloud-init/pull/4167

### CODE REVIEW

**Comment by aciba90** [comment]
After approval, I will replicate this in:

- [ ] lunar
- [ ] focal

**Review by blackboxsw — CHANGES_REQUESTED** [review]
@aciba90 thanks, two minor notes, otherwise +1

File: debian/changelog [hunk 1]
```diff
@@ -6,8 +6,10 @@ cloud-init (23.2-0ubuntu0~22.04.2) UNRELEASED; urgency=medium
     - Retain original groups in cloud.cfg.tmpl
   * Upstream snapshot based on upstream/main at 5509b790.
   * d/control: Add python3-passlib as needed for testing
+  * d/cloud-init.templates: enable Akamai by default.
+    Add Akamai to the default templates as enabled by default.
```
[the remark this answers is not in the document]
Comment (aciba90) [reply debian/changelog]: Applied, thanks.

File: debian/cloud-init.templates [hunk 2]
```diff
@@ -1,8 +1,8 @@
...
+__Choices: NoCloud: Reads info from /var/lib/cloud/seed only, ConfigDrive: Reads data from Openstack Config Drive, OpenNebula: read from OpenNebula context disk, DigitalOcean: reads data from Droplet datasource, Azure: read from MS Azure cdrom. Requires walinux-agent, AltCloud: config disks for RHEVm and vSphere, OVF: Reads data from OVF Transports, MAAS: Reads data from Ubuntu MAAS, GCE: google compute metadata service, OpenStack: native openstack metadata service, CloudSigma: metadata over serial for cloudsigma.com,  SmartOS: Read from SmartOS metadata service, Bigstep: Bigstep metadata service, Scaleway: Scaleway metadata service, AliYun: Alibaba metadata service, Ec2: reads data from EC2 Metadata service, CloudStack: Read from CloudStack metadata service, Hetzner: Hetzner Cloud, IBMCloud: IBM Cloud. Previously softlayer or bluemix., Oracle: Oracle Compute Infrastructure, Exoscale: Exoscale, RbxCloud: HyperOne and Rootbox platforms, UpCloud: UpCloud, VMware: reads data from guestinfo table or env vars, Vultr: Vultr Cloud, LXD: Reads /dev/lxd/sock representation of instance data, NWCS: NWCS, Akamai: Akamai, None: Failsafe datasource
```
[the remark this answers is not in the document]
Comment (aciba90) [reply debian/cloud-init.templates]: Applied, thanks.

**Comment by aciba90** [comment]
> @aciba90 thanks, two minor notes, otherwise +1

Applied, thanks!

**Review by blackboxsw — APPROVED** [review]
LGTM, @aciba90, you may need to rebase on upstream/ubuntu/focal before you push to upstream as I just pushed the separate PR fixing the daily build recipe too.

**Pull request merged**